### DISCUSSION TITLE
ATOM

### DISCUSSION THREAD

**Discussion by IMRS1311** [community/community#49251] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am new in the field and I can't declare that I'm expert by any means. but I got disappointed when I came to know that ATOM is stopped due to security issues. 
Ok fine. but my question is that if the response to security issues is to stop the systems, we would end up with no systems.
I believe it would be better to enhance the security and seal the gaps rather than abandon the system.

another valid question. what if similar security issues discovered in Github?

can anybody answer me?
Thanks

**Comment by samuel-ma** [comment]
@IMRS1311 
Atom was not discontinued due to security reasons
Microsoft already had visual studio and the entire suite was popular with developers eg the VS code desktop application
The focus of the entire microsoft team has been on those products
And to answer you question, any security issues discovered on github are quickly patched up
There's a great team behind it, that's y lots of developers choose to use Github

Here's a release from Microsoft on Atom

**As a desktop application, Atom just had no viable future in today's cloud-focused computing world, with GitHub yesterday (June 8) saying "we've decided to retire Atom in order to further our commitment to bringing fast and reliable software development to the cloud via Microsoft Visual Studio Code and GitHub Codespaces**

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @IMRS1311! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
